### PR TITLE
Add openPaymentSetup method to PaymentRequest [MITM-831]

### PR DESCRIPTION
**Pull request by trcoffman**
Expose the openPaymentSetup API to users of react-native-payments.

### CODE REVIEW

File: docs/NativePayments.md [hunk 1]
```diff
@@ -159,6 +159,22 @@ NativePayments
 
 ---
 
+### openPaymentSetup()**
```
Comment (hojason117) [inline docs/NativePayments.md]: Is the `**` intentional ?

File: docs/PaymentRequest.md [hunk 2]
```diff
@@ -114,6 +114,22 @@ PaymentRequest
 
 ---
 
+### openPaymentSetup()**
```
Comment (hojason117) [inline docs/PaymentRequest.md]: Same here.

**Review by hojason117 — APPROVED** [review]

File: js/PaymentRequest/__mocks__/index.js [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
-  }
+    addListener: () => {},
+  },
 };
 
 const mockReactNativeAndroid = Object.assign({}, mockReactNativeIOS, {
   Platform: {
-    OS: 'android'
-  }
+    OS: "android",
```
Comment (Karthz) [inline js/PaymentRequest/__mocks__/index.js:13]: is this change intentional?

**Review by Karthz — APPROVED** [review]
